[0m10:09:47.902547 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002560937FA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025608D5F170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025608D5EA20>]}


============================== 10:09:47.902547 | 7b233ea6-2038-4089-bf73-8efa4917e139 ==============================
[0m10:09:47.902547 [info ] [MainThread]: Running with dbt=1.9.3
[0m10:09:47.902547 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\TumpudiKarthikeya\\.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:10:08.764565 [debug] [MainThread]: Starter project path: C:\Users\TumpudiKarthikeya\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m10:10:08.914919 [info ] [MainThread]: 
Your new dbt project "project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:10:08.916936 [info ] [MainThread]: Setting up your profile.
[0m10:12:33.959322 [info ] [MainThread]: Profile project written to C:\Users\TumpudiKarthikeya\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m10:12:33.962450 [debug] [MainThread]: Command `dbt init` succeeded at 10:12:33.962450 after 166.29 seconds
[0m10:12:33.963460 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256095E0A40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256160FE930>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025615C422A0>]}
[0m10:12:33.963839 [debug] [MainThread]: Flushing usage events
[0m10:12:35.034001 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:52:05.144649 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60C32F140>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60C32D8B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60A099430>]}


============================== 11:52:05.144649 | a3c77284-3ac2-4603-a087-974beb9ebbf7 ==============================
[0m11:52:05.144649 [info ] [MainThread]: Running with dbt=1.9.3
[0m11:52:05.144649 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\TumpudiKarthikeya\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt run -m tag:staging', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:52:05.144649 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\TumpudiKarthikeya\Desktop\mini_project_2\dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:52:05.161214 [debug] [MainThread]: Command `dbt run` failed at 11:52:05.161214 after 0.26 seconds
[0m11:52:05.161603 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60C67A660>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60C32D8B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E60C32F140>]}
[0m11:52:05.161603 [debug] [MainThread]: Flushing usage events
[0m11:52:06.699331 [debug] [MainThread]: An error was encountered while trying to flush usage events
